### PR TITLE
UI to update ticket status in the ticket header

### PR DESCRIPTION
**Pull request by danieljames-dj**
Currently tickets UI doesn't allow to change ticket status. This PR aims at having that feature.

One of the use-case is that if WRT decides not to take any action for a particular ticket, there should be option to change the status to closed.

### CODE REVIEW

File: app/webpacker/components/Tickets/TicketHeader/StatusView.jsx
Comment (gregorbg) [inline app/webpacker/components/Tickets/TicketHeader/StatusView.jsx:48]: Not a fan of computing these inline. Move to a separate `useMemo` please :upside_down_face:

**Review by gregorbg — DISMISSED** [review]

File: app/webpacker/components/Tickets/TicketHeader/StatusView.jsx
Comment (danieljames-dj) [reply app/webpacker/components/Tickets/TicketHeader/StatusView.jsx]: Done

File: app/webpacker/components/Tickets/TicketHeader/StatusView.jsx
Comment (kr-matthews) [inline app/webpacker/components/Tickets/TicketHeader/StatusView.jsx:75]: Just one very minor suggestion - the child doesn't need the full setter:
```suggestion
      onCancel={() => setEditMode(false)}
```

Comment (danieljames-dj) [reply app/webpacker/components/Tickets/TicketHeader/StatusView.jsx]: Done, thank you.

**Review by kr-matthews — APPROVED** [review]

File: app/webpacker/components/Tickets/TicketHeader/StatusView.jsx
Comment (danieljames-dj) [reply app/webpacker/components/Tickets/TicketHeader/StatusView.jsx]: Since this is minor front-end only change, I think it's fine to go ahead without Gregor's approval. The merging is blocked because Gregor has 'requested for a change'. I think only option for me is to skip Gregor's approval is to dismiss the review. Since I have already actioned what Gregor requested, I'm going ahead with dismissing Gregor's review. Hope this works.

**Pull request merged**
